src/Features/CSharp/Portable/Completion/KeywordRecommenders/UsingKeywordRecommender.cs

```diff
@@ -19,6 +19,7 @@ protected override bool IsValidContext(int position, CSharpSyntaxContext context
             //  using (goo) { }
             //  using Goo;
             //  using Goo = Bar;
+            //  await using (goo) { }
             return
                 context.IsStatementContext ||
                 context.IsGlobalStatementContext ||
```

Comment:
this comment was updated... but it's unclear to me how the code actually handles htis case.

Reply:
After an `await` we're in "statement context". I agree it's coarse.
I thought about adding a new value on context that would be narrower, but I'm not sure that's worthwhile. Previously you'd told me it's better to offer a superfluous recommendation than not offer a necessary one.
Let me know what you think.